steps/utility/utility.go

```diff
@@ -42,8 +42,7 @@ var _ = gauge.Step("Switch to project <projectName>", func(projectName string) {
 	gauge.GetScenarioStore()["namespace"] = projectName
 })
 
-var _ = gauge.Step("Validate route url for pipelines tutorial", func() {
-	expectedOutput := "Cat 🐺 vs Dog 🐶"
+var _ = gauge.Step("Validate route url which contains <expectedOutput>", func(expectedOutput string) {
 	routeUrl := store.GetScenarioData("routeurl")
 	output := cmd.MustSuccedIncreasedTimeout(30*time.Second, "lynx", routeUrl, "--dump").Stdout()
 	log.Println(output)
```

Comment:
Since the page content is very long, IMHO we could only print it when it doesn't contain the expected string, i.e. move line 48 inside the following `if` statement

Reply:
Done